### PR TITLE
sample-game-server document bug fix

### PR DESCRIPTION
**Pull request by hellof20**
What type of PR is this?
/kind documentation

What this PR does / Why we need it:
Change UNHEATHY to UNHEALTHY

Special notes for your reviewer:

### CODE REVIEW

**Comment by markmandel** [comment]
Good catch! That's been there for YEARS.

**Comment by markmandel** [comment]
Looks like a flake in the nodejs conformance tests

```
Watch ERROR Error: 14 UNAVAILABLE: Connection dropped
    at Object.callErrorFromStatus (/go/src/agones.dev/agones/sdks/nodejs/node_modules/@grpc/grpc-js/build/src/call.js:31:19)
    at Object.onReceiveStatus (/go/src/agones.dev/agones/sdks/nodejs/node_modules/@grpc/grpc-js/build/src/client.js:360:49)
    at Object.onReceiveStatus (/go/src/agones.dev/agones/sdks/nodejs/node_modules/@grpc/grpc-js/build/src/client-interceptors.js:328:181)
    at /go/src/agones.dev/agones/sdks/nodejs/node_modules/@grpc/grpc-js/build/src/call-stream.js:188:78
    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)
for call at
    at ServiceClientImpl.makeServerStreamRequest (/go/src/agones.dev/agones/sdks/nodejs/node_modules/@grpc/grpc-js/build/src/client.js:343:34)
    at ServiceClientImpl.watchGameServer (/go/src/agones.dev/agones/sdks/nodejs/node_modules/@grpc/grpc-js/build/src/make-client.js:105:19)
    at AgonesSDK.watchGameServer (/go/src/agones.dev/agones/sdks/nodejs/src/agonesSDK.js:132:30)
    at connect (/go/src/agones.dev/agones/test/sdk/nodejs/testSDKClient.js:25:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 14,
  details: 'Connection dropped',
  metadata: Metadata { internalRepr: Map(0) {}, options: {} }
}
make[2]: *** [includes/sdk.mk:145: run-sdk-conformance-no-build] Error 1
```

Weird that it didn't seem to succeed on retry (did it retry?)

**Comment by markmandel** [comment]
Just noting - once you sign the CLA, I'll kickoff the job again, and I expect it will pass. That's an odd flake.

**Comment by markmandel** [comment]
Just touching base on this update, seeing if you had time to submit the CLA?

**Comment by markmandel** [comment]
Just gently bumping this - if you can get the CLA sorted, we'd be happy to take this submission!

**Review by gongmax — APPROVED** [review]

File: examples/simple-game-server/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ There are some text commands you can send the server to affect its behavior:
 | Command             | Behavior                                                                                 |
 | ------------------- | ---------------------------------------------------------------------------------------- |
 | "EXIT"              | Causes the game server to exit cleanly calling `os.Exit(0)`                              |
-| "UNHEATHY"          | Stopping sending health checks                                                           |
+| "UNHEALTHY"          | Stopping sending health checks                                                           |
```
Comment (gongmax) [inline examples/simple-game-server/README.md:20]: Please fix the indent before "|"

**Comment by markmandel** [comment]
Closing since this has been a while. Submitting a separate PR to reformat the tables as well.

**Pull request closed without merging**